api/src/main/java/com/github/retrooper/packetevents/event/EventManager.java

```diff
@@ -20,6 +20,7 @@
 
 import com.github.retrooper.packetevents.PacketEvents;
 import com.github.retrooper.packetevents.exception.InvalidHandshakeException;
+import com.github.retrooper.packetevents.protocol.packettype.PacketType;
```

Comment:
any reason for this random import being added here?

Reply:
My bad